openslides/motions/static/js/motions/csv.js

```diff
@@ -191,14 +205,23 @@ angular.module('OpenSlidesApp.motions.csv', [])
                         //row.push('"' + p_new.text.html + '"');
 
                         // Work around: Export the full paragraphs instead of changed lines
-                        row.push('"' + amendment.getAmendmentParagraphsByMode('original', null, false)[0].text + '"');
-                        row.push('"' + amendment.getAmendmentParagraphsByMode('changed', null, false)[0].text + '"');
+                        // Remove all HTML tags from old and new text
+                        var oldText = document.createElement("DIV");
+                        oldText.innerHTML =  amendment.getAmendmentParagraphsByMode('original', null, false)[0].text;
+                        var newText = document.createElement("DIV");
+                        newText.innerHTML =  amendment.getAmendmentParagraphsByMode('changed', null, false)[0].text;
+                        row.push('"' + (oldText.textContent || oldText.innerText) + '"');
+                        row.push('"' + (newText.textContent || newText.innerText) + '"');
```

Comment:
Update: I used textContent/innerText for striping html tags and replacing special chars. - Rebased.